itables/datatables_format.py

```diff
@@ -13,7 +13,11 @@ def _format_column(x):
     if dtype_kind in ["b", "i", "s"]:
         return x
 
-    x = fmt.format_array(x.values, None, justify="all", leading_space=False)
+    try:
+        x = fmt.format_array(x._values, None, justify="all", leading_space=False)
+    except TypeError:
+        # Older versions of Pandas don't have 'leading_space'
+        x = fmt.format_array(x._values, None, justify="all")
```

Comment:
It might be an option to increase the coverage rate by using a combination of:

1. [assertions-about-expected-exceptions](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions) 
2. [skipping-if](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1)

for testing the python version dependency.

Reply:
Hi @Anselmoo , I think in this particular case the two cases are covered on the CI (as we do test against Pandas in version 0.22 which misses the `leading_space` argument).

However I see that the overall coverage is not so great, because we don't seem to have tests on the "column filters". Later on I may simply add a test that runs all the examples in the documentation, that will certainly help increasing the coverage...

Reply:
Actually that has been simpler than expected: https://github.com/mwouts/itables/pull/132